pkg/standalone-utils/contracts/CowSwapFeeBurner.sol

```diff
@@ -64,18 +67,18 @@ contract CowSwapFeeBurner is ICowSwapFeeBurner, ERC165, SingletonAuthentication
     ***************************************************************************/
 
     /// @inheritdoc ICowSwapFeeBurner
-    function getOrder(IERC20 sellToken) external view returns (GPv2Order memory) {
-        return _getOrder(sellToken);
+    function getOrder(IERC20 tokenIn) external view returns (GPv2Order memory) {
+        return _getOrder(tokenIn);
     }
 
     /// @inheritdoc ICowSwapFeeBurner
-    function getOrderStatus(IERC20 sellToken) external view returns (OrderStatus status) {
-        (status, ) = _getOrderStatusAndBalance(sellToken);
+    function getOrderStatus(IERC20 tokenIn) external view returns (OrderStatus status) {
+        (status, ) = _getOrderStatusAndBalance(tokenIn);
     }
 
     /// @inheritdoc ICowSwapFeeBurner
-    function retryOrder(IERC20 sellToken, uint256 minTargetTokenAmount, uint256 deadline) external authenticate {
-        (OrderStatus status, uint256 amount) = _getOrderStatusAndBalance(sellToken);
+    function retryOrder(IERC20 tokenIn, uint256 minTargetTokenAmount, uint256 deadline) external authenticate {
```

Comment:
```suggestion
    function retryOrder(IERC20 tokenIn, uint256 minAmountOut, uint256 deadline) external authenticate {
```
Consider renaming this one too? There are still some targetTokenAmounts out there; should we rename them to match the other renames? Here and elsewhere.